src/childHealth/utils.py

```
@@ -0,0 +1,44 @@
+import pandas as pd
+import yaml  # Fixed the import statement
+from childHealth.config import ProjectConfig
+import matplotlib.pyplot as plt
+import numpy as np
+
+def visualize_results(y_test, y_pred):
+    """
+    Visualize the results of predictions against actual values.
+    
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)